singer/src/main/java/com/pinterest/singer/common/SingerSettings.java

```diff
@@ -196,6 +207,18 @@ public static void initialize(SingerConfig config)
     globalFsm.start();
   }
 
+  protected static void loadAndSetLoggingAuditClient(SingerConfig config) {
```

Comment:
shall we name the method `loadAndSetLoggingAuditClientIfEnabled`?

Reply:
made the change in the update of the PR